src/linagora.esn.james/app/common/james-api-client.service.spec.js

```diff
@@ -6,7 +6,7 @@ const jamesApi = require('esn-api-client/src/api/james');
 const { expect } = chai;
 
 describe('The jamesApiClient factory', function() {
-  let jamesApiClient;
+  let jamesApiClient; // eslint-disable-line no-unused-vars
```

Comment:
Why is a comment added and not the line removed if it is unused?

Reply:
it's used after with also a `// eslint-disable-line no-unused-vars` and without the `jamesApiClient` mock the test didn't work so it's a quick fix